oshi-core/src/main/java/oshi/util/ParseUtil.java

```diff
@@ -1344,7 +1344,7 @@ public static String removeLeadingDots(String dotPrefixedStr) {
                 }
                 // Otherwise add string and reset start
                 // Intentionally using platform default charset
-                strList.add(new String(bytes, start, end - start));
+                strList.add(new String(bytes,StandardCharsets.UTF_8));
```

Comment:
You still need to limit the string length with `start, end-start`.  You'll ad the charset as a fourth parameter.  Use this constructor:  https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#%3Cinit%3E(byte%5B%5D,int,int,java.nio.charset.Charset)